src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Payment/PaymentReversalTests.cs

```diff
@@ -1,87 +1,96 @@
+                .Actions.Rows[y => y.Name.Value.Contains(PaymentResourceOperations.CreateReversal)].ExecuteAction.ClickAndGo()
+                .Actions.Rows[y => y.Name.Value.Contains(PaymentResourceOperations.PaidPayment)].Should.BeVisible()
+                .Actions.Rows[y => y.Name.Value.Contains(PaymentResourceOperations.ViewPayment)].Should.BeVisible()
+                .Actions.Rows.Count.Should.Equal(2);
+
+            var order = await SwedbankPayClient.Payments.CardPayments.Get(paymentLink, PaymentExpand.All);
+
+            // Operations
+            Assert.That(order.Operations[LinkRelation.CreateCancellation], Is.Null);
+            Assert.That(order.Operations[LinkRelation.CreateCapture], Is.Null);
+            Assert.That(order.Operations[LinkRelation.CreateReversal], Is.Null);
+            Assert.That(order.Operations[LinkRelation.PaidPayment], Is.Not.Null);
+
+            // Transactions
+            Assert.That(order.PaymentResponse.Transactions.TransactionList.Count, Is.EqualTo(3));
+            Assert.That(order.PaymentResponse.Transactions.TransactionList.First(x => x.Type == TransactionType.Authorization).State,
+                        Is.EqualTo(State.Completed));
+            Assert.That(order.PaymentResponse.Transactions.TransactionList.First(x => x.Type == TransactionType.Capture).State,
+                        Is.EqualTo(State.Completed));
+            Assert.That(order.PaymentResponse.Transactions.TransactionList.First(x => x.Type == TransactionType.Reversal).State,
+                        Is.EqualTo(State.Completed));
+        }
+
+
+        [Test]
+        [Retry(3)]
+        [TestCaseSource(nameof(TestData), new object[] { false, PaymentMethods.Swish })]
+        public async Task Payment_Swish_Reversal(Product[] products, PayexInfo payexInfo)
+        {
+            GoToOrdersPage(products, payexInfo, Checkout.Option.LocalPaymentMenu)
+                .PaymentLink.StoreValue(out var paymentLink)
+                .Actions.Rows[y => y.Name.Value.Contains(PaymentResourceOperations.CreateReversal)].ExecuteAction.ClickAndGo()
+                .Actions.Rows[y => y.Name.Value.Contains(PaymentResourceOperations.ViewPayment)].Should.BeVisible()
+                .Actions.Rows.Count.Should.Equal(1);
+
+            var swishPayment = await SwedbankPayClient.Payments.SwishPayments.Get(paymentLink, PaymentExpand.All);
+            var counter = 0;
+
+            while (swishPayment.PaymentResponse.Transactions.TransactionList.First(x => x.Type == TransactionType.Reversal).State != State.Completed && counter <= 15)
+            {
+                Thread.Sleep(1000);
+                try
+                {
+                    swishPayment = await SwedbankPayClient.Payments.SwishPayments.Get(paymentLink, PaymentExpand.All);
+                }
+                catch (HttpResponseException)
+                {

```

Comment:
Codacy found an issue: [Either remove or fill this block of code.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5523315)